src/MenuItem.react.js

```diff
@@ -33,7 +33,7 @@ const MenuItem = React.createClass({
           'active': active,
           'disabled': disabled,
         }, className)}>
-        <a href="#" onClick={this._handleClick}>
+        <a style={{ cursor: 'pointer' }} onClick={this._handleClick}>
```

Comment:
I'd need to test, but using `role="button"` would provide the same result and might be a little more semantic.